build.gradle

```diff
@@ -18,9 +18,9 @@ repositories {
 }
 
 dependencies {
-    compile "org.openjfx:javafx-base:11-ea+25:${platform}"
-    compile "org.openjfx:javafx-graphics:11-ea+25:${platform}"
-    compile "org.openjfx:javafx-controls:11-ea+25:${platform}"
+    compile "org.openjfx:javafx-base:11:${platform}"
+    compile "org.openjfx:javafx-graphics:11:${platform}"
+    compile "org.openjfx:javafx-controls:11:${platform}"
```

Comment:
I presume you've tested this, and that `11` works and doesn't need to be `11.0.0`?

Reply:
This is a direct result from the fact that we use the short version for FCS builds. See PR https://github.com/javafxports/openjdk-jfx/pull/177

This way, the maven version will always reflect the javafx.version property.

Reply:
And here is a link to search.maven.org containing the 11 version of org.openjfx:javafx artifacts: https://search.maven.org/search?q=g:org.openjfx%20a:javafx*%20v:11